slack-dark-mode.ps1

```diff
@@ -6,6 +6,12 @@ Param(
     [switch] $ForceSonicProtection
 )
 
+$default_npm_cache = &npm config get cache --global
+If ($default_npm_cache -match " "){
+    # if the path contains a space, then lets set to a temporary location until we finish.
+    &npm config set cache C:\tmp\nodejs\npm-cache --global

```

Comment:
Move this after the `npx` check below, that way we don't try to mess with the cache if the nodejs tools aren't installed.

Reply:
good point